assets/less/hel/question-page.less

```diff
@@ -0,0 +1,35 @@
+.section-container {
+  .section-browser {
+    display: flex;
+    align-items: center;
+
+
+    & > * {
+      display: block;
+    }
+
+    .to-hearing {
+      flex-grow: 1;
+    }
+
+    .previous, .next {
+      margin: auto 5px;
+      padding: 5px;
+      border: 1px solid @link-color;
+
+      .type-name {
+        text-transform: lowercase;
+      }
+
+      &:hover {
+        border-color: @link-hover-color;
```

Comment:
Let's change the "currently viewed" color to grey, while keeping the hover color purple. This is more in line with the currently selected language color in the sidebar (also grey).
EDIT: put the comment in the wrong place. I mean the sidebar "currently viewed" section color.